include/lbann/utils/im2col.hpp

```diff
@@ -113,6 +113,7 @@ void col2im(const Mat& col,
  *  one. im2col will automatically call this routine if it detects a
  *  1x1 im2col.
  */
+template<El::Device Dev>
```

Comment:
I don't think we should template our im2col routines since we only expect to use them for CPU data.

Reply:
Agree, fixed